oops/oops.go

```diff
@@ -116,6 +135,7 @@ func Frames(err error) [][]Frame {
 	}
 
 	parsedStacks := make([][]Frame, 0, len(stacks))
+	skippedFramesInStack := make([]bool, 0, len(stacks))
 
 	// Walk the set of stacks backwards, starting with stack most closest to the
```

Comment:
Nit: while you're in here, "starting with the stack closest to the causal error"